### ISSUE TITLE
What is the correct parserOptions for IE11 compatibility checking?

### ISSUE THREAD

**Issue by foucdeg** [amilajack/eslint-plugin-compat#393]
This is my browserlist:

`"browserslist": [">0.2%", "not dead", "not ie <= 10", "not op_mini all"]`

It includes IE11.

Here is my .eslintrc :

```
{
  "extends": ["plugin:compat/recommended"],
  "env": {
    "browser": true
  },
  "parserOptions": { "ecmaVersion": 6 }
}
```

With this config, **arrow functions do not raise errors**.

If I drop `"parserOptions": { "ecmaVersion": 6 }`, the parser defaults to 5, which raises parsing errors for `let` and `const`:

- `let` raises "unexpected token [variable name]"
- `const` raises "Parsing error: The keyword 'const' is reserved"

I need eslint and eslint-plugin-compat to raise when an arrow function is used (not IE11 compatible), and not raise when `let` and `const` are used (IE11 supports them). What is the correct setup?

**Comment by amilajack** [comment]
Currently we don't lint for syntax errors. This is something that we're currently looking into.

**Comment by foucdeg** [comment]
@amilajack In this case I am not. I have a project with both a modern React SPA with Babel, and also legacy SSR views with html templates that include plain old Javascript files. My intention is to check the latter for IE11 compatibility.
